vroom/main.vroom

```diff
@@ -185,8 +220,8 @@ them are available, the message will show a line for each.
   :let fake_format2 = copy(fake_format)
   :let fake_format2.name = 'fake-format2'
   :let fake_format2.setup_instructions = 'LMGTFY'
-  :call codefmtlib#AddFormatter(fake_format2)
+  :call maktaba#extension#GetRegistry('codefmt').AddExtension(fake_format2)
 
   :FormatCode
-  ~ Formatter "fake-format" is not available. Setup instructions: RTFM
   ~ Formatter "fake-format2" is not available. Setup instructions: LMGTFY
+  ~ Formatter "fake-format" is not available. Setup instructions: RTFM
```

Comment:
Oh, right, forgot we did change precedence. So to be clear, last registered wins and we're just using "first-wins" to describe the recommended handling of the list from GetFormatters, right? Still makes sense to me, it's just subtle enough I keep forgetting.